### PR TITLE
1.2.0 Feature: Realtime: Channel Message History

### PR DESCRIPTION
**Pull request by QuintinWillison**
One of a family of pull requests addressing #3.

### CODE REVIEW

File: sdk.yaml [hunk 1]
```diff
@@ -248,6 +248,15 @@ Realtime:
       .synopsis: |
         The option to configure a `cipher`, specifying encryption algorithm and attributes.
         Used for subscribe and publish on this channel.
+    History:
+      .documentation:
+        - https://ably.com/docs/realtime/history
+      .specification: [RTL10, CP2a, RTL15a]
+      .synopsis: |
+        Used to return continuous message history up to the exact point a realtime channel was attached,
+        combining both instantaneous “live” history as well as the longer term
```
Comment (lmars) [inline sdk.yaml:257]: The phrase `instantaneous “live” history` doesn't sound quite right, are we really just explaining that this includes both ephemeral (last two minutes) and persisted (last 24-72 hours) history?

Comment (tbedford) [reply sdk.yaml:257]: Perhaps: "Used to return continuous message history up to the exact point a realtime channel was attached. This includes both messages persisted for the default duration of two minutes, and messages persisted for 24-72 by enabling the `persist all messages` channel rule."

Comment (QuintinWillison) [reply sdk.yaml]: Thanks, @tbedford, I've used your words in [6a1685e](https://github.com/ably/features/pull/40/commits/6a1685e0837d1c2ab01c7b3b8241cf4ca2fa5120).

I'm now going to resolve this conversation on @lmars' behalf, in order to allow me to merge this pull request more quickly.

**Comment by QuintinWillison** [comment]
Having addressed @lmars' comment, with help, I'm going to merge this pull request without waiting for approval. It has been open for a while and there is a stack of pull requests ahead of it. We are likely to have another look at the entire work at the end, to double check that it congeals holistically, so the risk of merge without approval is low.

**Pull request merged**